src/http/server.cr

```diff
@@ -148,7 +148,7 @@ class HTTP::Server
 
   # Creates a new HTTP server with a handler chain constructed from the *handlers*
   # array and the given block.
-  def self.new(handlers : Array(HTTP::Handler), &handler : HTTP::Handler::HandlerProc) : self
+  def self.new(handlers : Indexable(HTTP::Handler), &handler : HTTP::Handler::HandlerProc) : self
```

Comment:
Why not the blockless overload below as well?

Reply:
Because I forgot about it 😅 , I'll update the MR.